GitExtensions.sln

```diff
@@ -694,8 +694,8 @@ Global
 		{2B9B62FA-3E0C-45C5-9287-68B69413C5DF}.ReleaseTC|Mixed Platforms.Build.0 = Release|Any CPU
 		{2D18BE89-D210-49EB-A9DD-2246FBB3DF6D}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{2D18BE89-D210-49EB-A9DD-2246FBB3DF6D}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{2D18BE89-D210-49EB-A9DD-2246FBB3DF6D}.Debug|Mixed Platforms.ActiveCfg = Debug|Any CPU
-		{2D18BE89-D210-49EB-A9DD-2246FBB3DF6D}.Debug|Mixed Platforms.Build.0 = Debug|Any CPU
+		{2D18BE89-D210-49EB-A9DD-2246FBB3DF6D}.Debug|Mixed Platforms.ActiveCfg = Release|Any CPU
+		{2D18BE89-D210-49EB-A9DD-2246FBB3DF6D}.Debug|Mixed Platforms.Build.0 = Release|Any CPU
```

Comment:
:memo: We should delete one of **Mixed Platforms** or **Any CPU** unless both are useful. Not good having multiple configurations when some are invalid. In this case, you only applied **Release** configuration to one of them.

Reply:
👍 Thank you. Yes, I totally forgot to apply it to the other configuration.

I'll do that for this PR and open an issue to discuss whether we can reduce down to a single configuration across the solution

Reply:
PR updated, and I created #5270 to track the configuration discussion.